specification/dimensions/billingcurrency.md

```diff
@@ -0,0 +1,34 @@
+# Billing Currency
+
+Billing Currency is an identifier that represents the currency that a charge for resources and/or services was billed in. Billing Currency is commonly used in scenarios where costs need to be grouped or aggregated.
+
+The BillingCurrency column MUST be present in the billing data. BillingCurrency MUST match the currency used in the invoice generated by the invoice issuer. This column must be of type String and MUST NOT contain null values. BillingCurrency MUST be one of the values supported in [ISO 4217:2015](https://www.iso.org/standard/64758.html).
+
+[Attribute: Currency Code Format](#currencycodeformat) contains further details about formatting requirements for currency values.
```

Comment:
```suggestion
[Attribute: Currency Code Format](#currencycodeformat) contains further formatting requirements for currency values.
```

Reply:
Is it possible to LINK to the Currency Code Format attribute?

Reply:
I believe that's what #currencycodeformat is supposed to link to.